lib/broccoli/angular2-app.js

```diff
@@ -91,6 +92,10 @@ Angular2App.prototype.toTree = function() {
       vendorNpmJs,
       thirdPartyJs
     ], { overwrite: true });
+
+    var manifest = new SwManifest(merged);
+
+    return mergeTrees([merged, manifest], { overwrite: true });
```

Comment:
Why overwrite? There's no file matching the appcache.manifest, and if there's one that should be an error.